### PR TITLE
feat(field-title): add description prop and fix css

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- Apps
  - [x] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

- Refactored `PFieldTitle` component to script setup.
- Added description to `PFieldTitle` component in mirinae package.
- Updated story of it.
image
image

- Fixed some css differences between original css and figma design. The left is new one and the right is the old(wrong) one.
image

Component spec in figma
image

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Review by skdud4659 — COMMENTED** [review]
LGTM-!!!! + wz little question:)

File: packages/mirinae/src/data-display/field-title/PFieldTitle.vue [hunk 1]
```diff
@@ -1,37 +1,47 @@
...
+            <slot>
+                {{ props.label }}
+            </slot>
+        </label>
+        <span v-if="props.description"
+              class="description"
+        >
+            {{ props.description }}
+        </span>
+    </span>
```
Comment (skdud4659) [inline packages/mirinae/src/data-display/field-title/PFieldTitle.vue:15]: Question:
Is there any specific reason why the span tag is wrapping the label tag?
As far as I know, the label tag is typically used with input elements, so if the purpose of the label is for specific styling, what do you think about applying the styles using CSS?

**Review by skdud4659 — APPROVED** [review]

**Review by sulmoJ — APPROVED** [review]
LGTM!

File: packages/mirinae/src/data-display/field-title/PFieldTitle.vue (hunk 1 above)
Comment (WANZARGEN) [reply packages/mirinae/src/data-display/field-title/PFieldTitle.vue]: No there is no reason. I think it just span tag is proper in this case!

File: packages/mirinae/src/data-display/field-title/PFieldTitle.vue [hunk 2]
```diff
@@ -2,11 +2,11 @@
     <span class="p-field-title"
           v-on="$listeners"
     >
-        <label>
+        <span class="title">
```
Comment (skdud4659) [inline packages/mirinae/src/data-display/field-title/PFieldTitle.vue:5]: thx!!!🥰

**Pull request merged**